jujugui/static/gui/src/app/components/generic-input/generic-input.js

```diff
@@ -213,21 +235,29 @@ YUI.add('generic-input', function() {
     },
 
     render: function() {
+      const errors = this.state.errors || this.props.errors;
```

Comment:
Is this conflating an array and a bool? Could you please make this logic more explicit?